cmd/template/app/flag.go

```diff
@@ -46,6 +48,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.Name, flagName, "", "Name of the app in the Catalog.")
 	cmd.Flags().StringVar(&f.Namespace, flagNamespace, "", "Namespace where the app will be deployed.")
 	cmd.Flags().StringVar(&f.Cluster, flagCluster, "", "Name of the cluster the app will be deployed to.")
+	cmd.Flags().StringVar(&f.Organization, flagOrganization, "", "Workload cluster organization.")
```

Comment:
I have a concern about the flag. As right now this will only work for CAPO clusters.

IIRC we hid the CAPO specific flags for `template cluster`. Should we do the same here until more providers are supported?

Reply:
Good point, this flag is indeed going to be ignored for non-CAPO clusters, so hiding it makes total sense.

Reply:
Ok, flag is hidden now